### ISSUE TITLE
Settings show even if not in use

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/maxlength#1]
When adding a new field, all the Maxlength settings show even if maxlength is not intended to be used on the field. Maybe it would be cleaner to add a checkbox indicating you want to use maxlength, then display the settings? (Manualcrop does this, for example)

![Screen Shot 2020-08-28 at 8 40 49 AM](https://user-images.githubusercontent.com/871421/91567321-90698280-e90a-11ea-995d-bd7f7d6348c3.jpg)

**Comment by laryn** [comment]
![Screen Shot 2020-09-03 at 4 21 54 PM](https://user-images.githubusercontent.com/871421/92173421-9f1bd200-ee01-11ea-894a-7bff0e6ba7bb.jpg)

**Comment by yorkshire-pudding** [comment]
@laryn Yes please.  - I came to this issue queue to suggest such a thing.

Should include all the fields highlighted in the image in first post; your latter image seems to leave the countdown message setting still visible.

**Comment by yorkshire-pudding** [comment]
@laryn - was the [image](https://github.com/backdrop-contrib/maxlength/issues/1#issuecomment-686771484) a mockup or does some draft code exist?

**Comment by laryn** [comment]
@yorkshire-pudding It was just a mockup created in browser.

The mentioned example, however, does very similar in Manual Crop:

 - https://github.com/backdrop-contrib/manualcrop/blob/1.x-1.x/manualcrop.admin.inc#L665-L671

**Comment by yorkshire-pudding** [comment]
I thought this might be quick and easy so had a little look at this, but struggled with determining what the initial value should be for sites that already have the module; I tried to determine it based on whether the `maxlength_js` field was greater than zero but couldn't figure out the best way.

**Comment by laryn** [comment]
I think that seems reasonable -- automatically having the box checked to display settings if the `maxlength_js` field has a value.
